lib/Sema/TypeCheckStorage.cpp

```diff
@@ -1978,6 +1989,7 @@ static AccessorDecl *createSetterPrototype(AbstractStorageDecl *storage,
       if (info.valueVar) {
         if (auto setter = info.valueVar->getOpaqueAccessor(AccessorKind::Set)) {
           asAvailableAs.push_back(setter);
+          addEnclosingScopeAvailability();
```

Comment:
I'm guessing you don't have to worry about this with the getter because a property wrapper's getter cannot be less available than the property wrapper type, but if the wrapper defines a `_modify` as well?